scripts/build_rln.sh

```diff
@@ -42,10 +42,13 @@ else
     # Build rln instead
     # first, check if submodule version = version in Makefile
     cargo metadata --format-version=1 --no-deps --manifest-path "${build_dir}/rln/Cargo.toml"
-    submodule_version=$(
-      cargo metadata --format-version=1 --no-deps --manifest-path "${build_dir}/rln/Cargo.toml" \
-        | jq -r '.packages[] | select(.name == "rln") | .version'
-    )
+
+    if [[ "$OSTYPE" == "msys" || "$OSTYPE" == "win32" ]]; then
+        submodule_version=$(cargo metadata --format-version=1 --no-deps --manifest-path "${build_dir}/rln/Cargo.toml" | sed -n 's/.*"name":"rln","version":"\([^"]*\)".*/\1/p')
+    else
+        submodule_version=$(cargo metadata --format-version=1 --no-deps --manifest-path "${build_dir}/rln/Cargo.toml" | jq -r '.packages[] | select(.name == "rln") | .version')
```

Comment:
Are the `sed` and `jq` available by default on Windows?

Reply:
I think msys built in contains bash and all the main *nix utilities... at least how I remember...

Reply:
Yes jq come along with msys.